go/oasis-test-runner/oasis/oasis.go

```diff
@@ -151,11 +151,16 @@ func (n *Node) Stop() error {
 	return n.stopNode()
 }
 
-// Restart kills the node, waits for it to stop, and starts it again.
-func (n *Node) Restart() error {
+// Restart kills the node, waits for it to stop, and starts it again after delay.
+func (n *Node) Restart(ctx context.Context, startDelay time.Duration) error {
```

Comment:
You could name this `RestartAfter` and then have the usual `Restart` just use a zero delay value.